src/elements/accordion/src/index.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+
+import React, { useState } from 'react'
+import { jsx, useThemeUI } from 'theme-ui'
+import { Icon } from '@uswitch/trustyle.icon'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  title: string,
+  isInitiallyOpen?: boolean
+}
+
+const Accordion: React.FC<Props> = ({
+  title,
+  isInitiallyOpen,
+  children
+}) => {
+  const { theme: { accordion: accordionTheme = {}, colors = {} } }: any = useThemeUI()
+  const [isOpen, setIsOpen] = useState(!!isInitiallyOpen)
```

Comment:
Would it be better to just give this a default value instead of casting to boolean here?